CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ A record of the changes made to `ALPS for Wordpress`.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+##[3.0.33]
+### Feature
+- Added 6 slide slider 
+
```

Comment:
Please don't edit the CHANGELOG.md file. These files will be edited after the merge and when I create the new release.